extensions/amp-story/0.1/amp-story.js

```diff
@@ -738,7 +738,8 @@ export class AmpStory extends AMP.BaseElement {
    */
   forceRepaintForSafari_() {
     const platform = Services.platformFor(this.win);
-    if (platform.isSafari() || platform.isIos()) {
+    // Force repaint is only needed when transitioning from invisible to visible
+    if (!this.isDesktop_() && (platform.isSafari() || platform.isIos())) {
```

Comment:
Tiny little nit, but this would look better as a series of short-circuit clauses.

```javascript
const platform = Services.platformFor(this.win);
if (!platform.isSafari() && !platform.isIos()) {
  return;
}
if (this.isDesktop()) {
  // Repaint is only needed when transitioning page from invisible to visible.
  return;
}
this.mutateElement...
```

Reply:
Done